README.md

```diff
@@ -1,5 +1,20 @@
-# doc-template
-document template
+# Provenance data model recommendation v1.0
```

Comment:
I would not put the version number to the README. With the first new commit, this is no longer version 1.0, but some unofficial version in between. Then the version number is quite misleading.

Reply:
the version is the one of the recommendation, the only official one. The readme is a copy of the IVOA document page page, not much to review here...

Reply:
see #3 ...